sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -1387,7 +1387,7 @@ public virtual void AmqpManagementLinkCreateComplete(string eventHubName)
         }
 
         /// <summary>
-        ///   Indicates that an <see cref="EventHubBufferedProducerClient" /> instance is about to begin processing events.
+        ///   Indicates that a <see cref="EventHubBufferedProducerClient" /> instance is about to begin processing events.
```

Comment:
Isn't "an" correct here?

Reply:
hrmm... yeah.   Looks like I was a bit more permissive with the find/replace than intended.